tests/baseline.json

```diff
@@ -1,18 +1,18 @@
 {
   "model": {
-    "pop_size_mean": 2665.2295081967213,
-    "pop_growth_rate_mean": 3.020452106890077,
-    "mcpr_mean": 10.652639202429542,
-    "maternal_mortality_ratio": 462.2496147919877,
-    "infant_mortality_rate": 48.03493449781659,
-    "crude_death_rate": 6.731740154830024,
-    "crude_birth_rate": 38.539212386401886,
-    "total_fertility_rate_mean": 5.930731074529737,
-    "asfr_mean": 141.784951086091,
-    "skyscrapers_mean": 2.0408163265306127,
-    "spacing_stats_mean": 2.6666666666666288,
-    "age_first_stats_mean": 20.638888888888868,
-    "method_counts_mean": 0.09999999999999999
+    "pop_size_mean": 2087.311475409836,
+    "pop_growth_rate_mean": 2.1125285618320095,
+    "mcpr_mean": 9.644953551047262,
+    "maternal_mortality_ratio": 0.0,
```

Comment:
Wondering about these changes - worried about the 0.0 maternal mortality ratio. Any ideas here, @robynstuart @emilydriano ?

Reply:
These results are from a Senegal sim with 1000 agents. Maternal mortality in 2020 is 0.00214, so I would expect we might see 1 death annually but there'd be plenty of years with no deaths, right?